README.md

```diff
@@ -1,42 +1,81 @@
 
-
 [**Source**](https://github.com/bloodmc/GDHooks)  
 [**Wiki**](https://github.com/bloodmc/GDHooks/wiki)  
 [**Support Me**](https://www.patreon.com/bloodmc)  
 [**GriefDefender**](https://www.spigotmc.org/resources/griefdefender.68900/)  
 
-GDHooks currently provides GriefDefender hooks for the following plugins:  
-
-* BlueMap
-* BossShopPro
-* ChestShop
-* CustomItems
-* DynamicShop
-* Dynmap
-* EliteMobs
-* InsaneShops
-* MMOItems
-* McMMO
-* MyPet
-* MythicMobs
-* QuickShop
-* RevoltCrates
+GriefDefender currently support following plugins:  
+
+#### Supported By GDHook
+
+ **Map**
+
+* [BlueMap]: [ [Github](https://github.com/BlueMap-Minecraft/BlueMap) | [Spigot](https://www.spigotmc.org/resources/83557/) | [Ore](https://ore.spongepowered.org/Blue/BlueMap) ]
+* [Dynmap]: [ [Github](https://github.com/webbukkit/dynmap) | [Spigot](https://www.spigotmc.org/resources/274/) ]
+* [Pl3xMap]: [ [Github](https://github.com/pl3xgaming/Pl3xMap) ]
+
+**Shop**
+
+* [BossShopPro]: [ [Spigot](https://www.spigotmc.org/resources/222/) | [Github](https://github.com/Blackixx/BossShopPro) ]
+* [ChestShop]: [ [Spigot](https://www.spigotmc.org/resources/51856/) | [Github](https://github.com/ChestShop-authors/ChestShop-3) ] 
+* [DynamicShop]: [ [Spigot](https://www.spigotmc.org/resources/65603/) | [Github](https://github.com/7sat/SSDynamicShop) ]
+* [InsaneShops(Paid)]: [ [Spigot](https://www.spigotmc.org/resources/67352/) ]
+* [QuickShop]: [ [Spigot](https://www.spigotmc.org/resources/62575/) | [Github](https://github.com/Ghost-chu/QuickShop-Reremake) ]
+* [ShopChest]: [ [Spigot](https://www.spigotmc.org/resources/11431/) | [Github](https://github.com/EpicEricEE/ShopChest) ]
 * Shop
-* ShopChest
-* SimpleClans
-* Slabbo
-* UltimateShops
+* [Slabbo]: [ [Spigot](https://www.spigotmc.org/resources/81368/) | [Github](https://github.com/sevn65/Slaboo) ]
+* [UltimateShops]: [ [Lite](https://www.spigotmc.org/resources/61048/) | [Premium](https://www.spigotmc.org/resources/64925/) ]
+
+**Misc**
+
+* [CustomItems(Paid)]: [ [Spigot](https://www.spigotmc.org/resources/36128/) ]
+* [EliteMobs]: [ [Spigot](https://www.spigotmc.org/resources/40090/) | [Github](https://github.com/MagmaGuy/EliteMobs) ]
+* [MMOItems(Paid)]: [ [Spigot](https://www.spigotmc.org/resources/39267/) ]
+* [McMMO]: [ [Spigot](https://www.spigotmc.org/resources/64348/updates) | [Github](https://github.com/mcMMO-Dev/mcMMO) ]
+* [MyPet]: [ [Spigot](https://www.spigotmc.org/resources/mypet.12725/) | [Github](https://github.com/xXKeyleXx/MyPet) ]
+* [MythicMobs]: [ [Free](https://www.spigotmc.org/resources/5702/) | [Premium](https://www.spigotmc.org/resources/58415/) ]
+* [RevoltCrates(Paid)]: [ [Spigot](https://www.spigotmc.org/resources/81681/) ]
+* [SimpleClans]: [ [Spigot](https://www.spigotmc.org/resources/71242/) | [Github](https://github.com/RoinujNosde/SimpleClans) ]
+
+#### Supported By Plugin Side
```

Comment:
This section can go on a wiki page. It shouldn't be in the README as it is not related to GDHooks.

Reply:
Actually, maybe it might be a good idea to have it all in one place. I'll think about it.